internal/network/command_dns_record.go

```diff
@@ -37,8 +38,33 @@ func (c *command) newDnsRecordCommand() *cobra.Command {
 	return cmd
 }
 
-func (c *command) addAccessPointFlag(cmd *cobra.Command) {
-	cmd.Flags().String("private-link-access-point", "", "PrivateLink access point.")
+func (c *command) addPrivateLinkAccessPointFlag(cmd *cobra.Command) {
+	cmd.Flags().String("private-link-access-point", "", "Private Link access point.")
```

Comment:
Any way we can improve this description or is it self-explanatory enough?

Reply:
Also unsure about this one...

cc: @syoung-cflt

Reply:
@kabraham13 Do you have any suggestions for this one?